client/package-lock.json

```diff
@@ -1085,92 +1511,244 @@
             }
         },
         "node_modules/@swc/core": {
-            "version": "1.5.0",
+            "version": "1.4.17",
```

Comment:
It would be better to fix this issue by going back to the last version of `package-lock.json` containing the pinned versions, then re-doing whatever change introduced the removal of the pins in the first place

I have the feeling the numerous change in version in this PR are due to the fact we fixed the issue by re-pinning the version from scratch (hence losing the old pins and there precise versions)